projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js

```diff
@@ -118,6 +110,20 @@ export default function ProductManagementInspectorControl( {
 								{ __( 'Add subscription', 'jetpack' ) }
 							</Button>
 						</PanelRow>
+						{ siteSlug && (
+							<>
+								<PanelRow>
+									<ExternalLink href={ `https://wordpress.com/earn/payments/${ siteSlug }` }>
+										{ __( 'Manage your subscriptions.', 'jetpack' ) }
```

Comment:
This will need to be updated after #23566 is merged.

Reply:
Yes, we already have an issue on our board.